.gitignore

```diff
@@ -19,3 +19,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea
```

Comment:
I'm hiding my ideas. ;)

On a more serious tone, I frequently use Jetbrains RubyMine while doing some pretty hardcore stuff like big refactors, debugging using a, well, debugger (usually I'm a `p` debugger) and it creates this frigging directory in the root of the project. =)